README.md

```diff
@@ -40,9 +41,153 @@ Spline aims to fill a big gap within the Apache Hadoop ecosystem. Spark jobs sho
 
 ---
 
-    Copyright 2017 ABSA Group Limited
+## Get Spline
+To get started, you need to get a minimal set of Spline's moving parts - 
+a server, an admin tool and a client Web UI to see the captured lineage.
+
+There are two ways how to do it:
+
+#### Download prebuild Spline artifacts from the Maven repo
+-   [```za.co.absa.spline:admin:0.4.0```](https://repo1.maven.org/maven2/za/co/absa/spline/admin/0.4.0/)
+-   [```za.co.absa.spline:rest-gateway:0.4.0```](https://repo1.maven.org/maven2/za/co/absa/spline/rest-gateway/0.4.0/) 
+-   [```za.co.absa.spline:client-web:0.4.0```](https://repo1.maven.org/maven2/za/co/absa/spline/client-web/0.4.0/) (optional)
+
+(REST Server and Web Client modules are also available as [Docker containers](https://hub.docker.com/u/absaoss))
+
+-or-
+
+#### Build Spline from the source code
+1.  Make sure you have JDK 8, Maven and NodeJS installed.
+
+1.  Get and unzip the Spline source code:
+    ```shell script
+    wget https://github.com/AbsaOSS/spline/archive/release/0.4.0.zip
+    unzip 0.4.0.zip
+    ```
+1.  Change the directory:
+    ```shell script
+    cd spline-release-0.4.0
+    ```
+1.  Run the Maven build:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[ordered-list-marker-value] Marker should be `4`, was `1`](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4568241)